apps/api/src/rest/routes/library-traffic.ts

```diff
@@ -0,0 +1,59 @@
+import { defaultHook } from "$hooks";
+import {
+  errorSchema,
+  libraryTrafficQuerySchema,
+  libraryTrafficSchema,
+  responseSchema,
+} from "$schema";
+import { LibraryTrafficService } from "$services";
+import { OpenAPIHono, createRoute } from "@hono/zod-openapi";
+import { database } from "@packages/db";
+
+const libraryTrafficRouter = new OpenAPIHono<{ Bindings: Env }>({ defaultHook });
+
+const libraryTrafficRoute = createRoute({
+  summary: "Retrieve latest library traffic data",
+  operationId: "libraryTraffic",
+  tags: ["Library Traffic"],
+  method: "get",
+  path: "/",
+  request: { query: libraryTrafficQuerySchema },
+  description: "Retrieves the latest traffic data for all floors or a specific floor.",
```

Comment:
I think the part about floor(s) is a little confusing for a new reader. Perhaps this description should briefly explain what library traffic is and leave it at that.